server/services/core.js

```diff
@@ -139,7 +139,9 @@ const createSitemapEntries = async () => {
   // Collection entries.
   await Promise.all(Object.keys(config.contentTypes).map(async (contentType) => {
     const excludeDrafts = config.excludeDrafts && strapi.contentTypes[contentType].options.draftAndPublish;
-    const pages = await noLimit(strapi.query(contentType), {
+    const current = config.contentTypes[contentType]['languages']['und']
```

Comment:
Please note that this may be a bug. I'm still pretty new to i18n in strapi so I'm unsure about the implementation here. I manually fetch from 'und' here when I probably shouldn't. I think there is a minor fix here but I was hoping a maintainer could lend a hand on this.

Reply:
I think this works best if the filters are not tied to a specific language, but rather be the same for all.

The reason for that is becasue we query all the pages of a specific collection in a single DB query (with `strapi.entityService`). Regardless of the language.
If we were to specifiy different filters per language, we would have to split that db query up to query per language. That would be a performace decrease and I would like to prevent that.

I've updated the PR so that you can only save the filters once and they'll be applied for all languages.